### PR TITLE
8.0 add account_group_invoice_lines module

### PR DESCRIPTION
**Pull request by alexis-via**
Please refer to README for a full description of the module.

I think this module is really usefull and many accountants will like to be able to configure Odoo to have fewer account move lines in their sale and purchase journals !

### CODE REVIEW

File: account_group_invoice_lines/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Account Group Invoice Lines',
+    'version': '1.1',
```
Comment (tafaRU) [inline account_group_invoice_lines/__openerp__.py]: Please update version according to https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers

File: account_group_invoice_lines/account.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+##############################################################################
+
+from openerp import models, fields, api
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    def inv_line_characteristic_hashcode(self, invoice_line):
+        '''When grouping per account, we remove the product_id from
```
Comment (clonedagain) [inline account_group_invoice_lines/account.py]: triple-""" is the norm I believe, and the first line should be a short description

File: account_group_invoice_lines/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Account Group Invoice Lines',
+    'version': '1.1.0',
```
Comment (tafaRU) [inline account_group_invoice_lines/__openerp__.py]: Please also add at the beginning the Odoo version. Thanks.

Comment (alexis-via) [inline account_group_invoice_lines/__openerp__.py]: Odoo will automatically concatenate the version of Odoo (8.0) with the version of the module (1.1.0). So, with the current value for the version key, Odoo will display "8.0.1.1.0" as module version in the web user interface.

So do you really want me to put the version of Odoo at the beginning of the version number ?

Comment (pedrobaeza) [inline account_group_invoice_lines/__openerp__.py]: Please see this comment that explain the why: https://github.com/OCA/maintainer-tools/pull/90#issue-97329423

**Comment by alexis-via** [comment]
@clonedagain @tafaRU 
I took into account all your remarks in my latested commit... do you approve the PR now ?

**Comment by alexis-via** [comment]
and @alnslang told me he has signed OCA's CLA yesterday.

**Comment by tafaRU** [comment]
Thanks, @alexis-via! Now LGTM :+1:

**Comment by cdubuit** [comment]
:+1:  bien

**Comment by sla-subteno-it** [comment]
:+1:

**Comment by sebastienbeau** [comment]
:+1:

**Pull request merged**